doorstop.c

```diff
@@ -85,32 +87,63 @@ void doorstop_init_mono_functions(void *handle)
     LOAD_METHOD(mono_set_assemblies_path);
     LOAD_METHOD(mono_image_open_from_data_with_name);
 
+    LOAD_METHOD(mono_object_to_string);
+    LOAD_METHOD(mono_string_to_utf8);
+
 #undef LOAD_METHOD
 }
 
+#define LOG(message)                        \
```

Comment:
I have no real idea how c works. 
So if anybody knows how to create a proper log-method, I am open for suggestions.